app/renderers/hyrax/renderers/attribute_renderer.rb

```diff
@@ -15,10 +15,7 @@
   def render
     markup = ''
 
-    # This is a temporary solution for prohibiting blank metadata fields from being displayed.
-    # Many fields with no assigned values are returning as [""] causing the field to render.
-    # TODO: Find the root cause of this issue are return the following line to its original state.
-    return markup if Array.wrap(values).first.blank? && !options[:include_empty]
+    return markup if values.blank? && !options[:include_empty]
```

Comment:
Is this still needed?

Reply:
I removed the bandaid that was added in the previous commit. I reverted it to the way the code was before.